controls/os_spec.rb

```diff
@@ -27,6 +27,7 @@
 
 shadow_group = 'root'
 shadow_group = 'shadow' if os.debian? || os.suse?
+container_execution = virtualization.role == 'guest' && virtualization.system =~ /^(lxc|docker)$/
```

Comment:
In a next iteration, we should probably capture this logic in our custom inspec resource